tidb-cloud/secure-connections-to-serverless-tier-clusters.md

```diff
@@ -1,50 +1,28 @@
+Before connecting to your serverless cluster, you can refer to [Password authentication](tidb-cloud-password-authentication.md) or [SSO authentication]() to log in to TiDB Cloud.
 
-However, some drivers and ORMs do not use the system root CA stores. In those cases, you need to configure the CA root path of the drivers or ORMs to your system root CA stores. For example, when you use [mysqlclient](https://github.com/PyMySQL/mysqlclient) to connect a TiDB Serverless Tier cluster in Python on macOS, you need to set `ca: /etc/ssl/cert.pem` in the `ssl` argument.
-
-> **Note:**
->
-> TiDB Serverless Tier does not provide a CA root certificate download, because we don't guarantee that the same CA will be used to issue a certificate in the future, which will cause the CA root certificate to change.
->
-> However, TiDB Serverless Tier ensures always using a CA root certificate that is commonly available, which is provided in all common systems.
->
-> If you really need the CA certificate of a TiDB Serverless Tier cluster, it is recommended that you download the [Mozilla CA Certificate bundle](https://curl.se/docs/caextract.html) instead of the single CA certificate in case we change the CA in the future.
->
-> If you are using a GUI client, such as DBeaver, which does not accept a certificate file with multiple certificates inside, you must download the [ISRG Root X1](https://letsencrypt.org/certs/isrgrootx1.pem.txt) certificate.
+We simplify the operation process of setting database passwords, allowing you to more conveniently and securely set and reset database cluster passwords on the connect page of your Serverless cluster.
 
-## How do I connect to a TiDB Serverless Tier cluster in TLS connection?
+## Secure connect to TiDB database cluster
 
-TiDB Cloud provides some connection examples in the **Connect** dialog. You can follow the instructions in [Connect via standard connection](/tidb-cloud/connect-via-standard-connection.md) to connect to a TiDB Serverless Tier cluster.
+we provides different connection examples in the **Connect** dialog. After creating your TiDB Serverless Cluster, click Cluster to see detailed cluster information, and you can see the "Connect" label in the upper right corner. After clicking, you will enter the function page for setting client, application and development tools to connect to the database.
+   - Support Mysql CLI, MyCLI, JDBC, Python, Go, Node.js and other six ways to connect to your Serverless cluster
```

Comment:
Does this mean we can only use these six ways to connect to Serverless cluster?

Reply:
maybe have more in the feature